xlgui/properties.py

```diff
@@ -80,10 +81,10 @@ def __init__(self, parent, tracks, current_position=0, with_extras=False):
         self.__default_attributes = Pango.AttrList()
         self.__changed_attributes = Pango.AttrList()
 
-        self.message = dialogs.MessageBar(
-            parent=self.builder.get_object('main_container'),
-            buttons=Gtk.ButtonsType.CLOSE
-        )
+        main_container = self.builder.get_object('main_container')
+        self.message = dialogs.MessageBar(buttons=Gtk.ButtonsType.CLOSE)
+        main_container.insert_row(0)
+        main_container.attach(self.message, 0, 0, 6, 1)
```

Comment:
Could you keep the old layout, with `main_container` being a VBox? That way if we add new buttons we don't need to update this.